.github/workflows/mocha.yml

```diff
@@ -0,0 +1,158 @@
+          - windows-2019
+        node:
+          # - 10
+          # - 12
+          - 14
+        include:
+          - os: ubuntu-latest
+            node: 14
+            env:
+              COVERAGE: 1
+    steps:
+      - name: Cache Growl Installer (Windows)
+        if: "${{ matrix.os == 'windows-2019' }}"
+        id: cache-growl
+        uses: actions/cache@v2
+        with:
+          path: GrowlInstaller
+          key: '${{ runner.os }}-growl-installer'
+          restore-keys: |
+            ${{ runner.os }}-growl-installer
+      - name: Download Growl Installer (Windows)
+        if: "${{ matrix.os == 'windows-2019' && steps.cache-growl.outputs.cache-hit != 'true'}}"
+        run: >
+          echo "Downloading Growl installer..."
+
+          mkdir GrowlInstaller | out-null
+
+          $seaURL =
+          "https://github.com/briandunnington/growl-for-windows/releases/download/final/GrowlInstaller.exe"
+
+          $seaPath = "GrowlInstaller\GrowlInstaller.exe"
+
+          $webclient = New-Object Net.WebClient
+
+          $webclient.DownloadFile($seaURL, $seaPath)
+
+          7z x $seaPath -oGrowlInstaller | out-null
+
+      - name: Retrieve Growl Installer (Windows)
+        if: "${{ matrix.os == 'windows-2019' }}"
+        uses: actions/cache@v2
+        with:
+          path: GrowlInstaller
+          key: '${{ runner.os }}-growl-installer'
+          restore-keys: |
+            ${{ runner.os }}-growl-installer
+      - name: Add Growl Installer to Path (Windows)
+        if: "${{ matrix.os == 'windows-2019' }}"
+        run: 'echo "::add-path::C:\Program Files (x86)\Growl for Windows"'
+      - name: Install Growl
+        if: "${{ matrix.os == 'windows-2019' }}"
+        run: |
+          echo "Installing Growl..."
+          cmd /c start /wait msiexec /i GrowlInstaller\Growl_v2.0.msi /quiet
+      - name: Start Growl Service (Windows)
+        if: "${{ matrix.os == 'windows-2019' }}"
+        run: |
+          echo "Starting Growl service..."
+          Start-Process -NoNewWindow Growl
+          ## Growl requires some time before it's ready to handle notifications
+          echo "Verifying Growl responding"
+          Start-Sleep -s 10
+          growlnotify test
+      - name: Install libnotify-bin (Linux)
+        if: "${{ matrix.os == 'ubuntu-latest' }}"
+        run: sudo apt-get install libnotify-bin
+
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '${{ matrix.node }}'
+      - name: Get npm cache directory
+        id: npm-cache
+        run: |
+          echo "::set-output name=dir::$(npm config get cache)"
+      - uses: actions/checkout@v2
+      - name: 'Cache node_modules'
+        uses: actions/cache@v2
+        with:
+          path: '${{ steps.npm-cache.outputs.dir }}'
+          key: "${{ matrix.os }}-node-v${{ matrix.node }}-${{ hashFiles('**/package-lock.json') }}"
+          restore-keys: |
+            ${{ matrix.os }}-node-v${{ matrix.node }}-
```

Comment:
When I test cache on windows before, it took about 80s to download over 600MB cache files.

In this GHA, "Install Dependencies" took like below without cache.  So, I worried about using cache takes more time.

### Ubuntu
```
Run npm ci --ignore-scripts
added 2827 packages in 24.523s
```
### Windows
``` 
Run npm ci --ignore-scripts
added 2827 packages in 65.224s
```

Reply:
I'll profile this and see if we should be caching or no

Reply:
turns out windows has a cache of like 600MiB, and linux is 50MiB, so that's why.

I changed this to use the cache for linux but not windows.